examples/images_contours_and_fields/contour_demo.py

```diff
@@ -57,7 +56,7 @@
 ###############################################################################
 # You can set negative contours to be solid instead of dashed:
 
-matplotlib.rcParams['contour.negative_linestyle'] = 'solid'
+#    - `matplotlib.rcParams['contour.negative_linestyle'] = 'solid'`
 fig, ax = plt.subplots()
 CS = ax.contour(X, Y, Z, 6, colors='k')  # Negative contours default to dashed.
```

Comment:
Is the comment on this line incorrect now the `rcParams` line is commented out?

Reply:
Sorry, search-replace error. Reverted.